src/cow-react/modules/limitOrders/utils/getOrderExecutedAmounts.ts

```diff
@@ -0,0 +1,28 @@
+import { Order } from 'state/orders/actions'
+import JSBI from 'jsbi'
+
+/**
+ * Syntactic sugar to get the order's executed amounts as a BigNumber (in atoms)
+ * Mostly because `executedSellAmount` is derived from 2 fields (at time or writing)
+ *
+ * @param order The order
+ */
+export function getOrderExecutedAmounts(order: Order): {
+  executedBuyAmount: JSBI
+  executedSellAmount: JSBI
+} {
+  if (!order.apiAdditionalInfo) {
+    return {
+      executedBuyAmount: JSBI.BigInt(0),
+      executedSellAmount: JSBI.BigInt(0),
+    }
+  }
+
+  return {
+    executedBuyAmount: JSBI.BigInt(order.apiAdditionalInfo.executedBuyAmount),
+    executedSellAmount: JSBI.subtract(
```

Comment:
Is this correct? `executedSellAmount = executedSellAmount - executedFeeAmount`

Just kidding, but let me explain my point matematically :P 

Given any `executedFeeAmount > 0`

`executedSellAmount = executedSellAmount - executedFeeAmount`

`executedSellAmount - executedSellAmount = executedFeeAmount`

`0 = executedFeeAmount` --> 🔥 false, because `executedFeeAmount > 0`

Reply:
So, should this be `executedSellAmountAfterFee` or sth?